Scroller/EditableWrapper.js

```diff
@@ -374,6 +400,32 @@ const EditableWrapper = (props) => {
 		}
 	}, [removeItem, editable.removeItemFuncRef]);
 
+	useEffect(() => {
+		// addEventListener to moveItems while scrolled
+		const scrollContentNode = scrollContentRef.current;
+		const scrollContentCenter = scrollContentNode.getBoundingClientRect().width / 2;
+
+		const handleMoveItemsByScroll = () => {
+			const {centeredOffset, itemWidth, mouseClientX, selectedItem} = mutableRef.current;
+			if (selectedItem) {
+				const toIndex = Math.floor((mouseClientX + scrollContentNode.scrollLeft - centeredOffset) / itemWidth);
```

Comment:
Do we need `centeredOffset` here? If it's "scrollable" then, I think it's meaningless.